ops/model.py

```diff
@@ -769,22 +763,22 @@ class Port:
 OpenedPort = Port  # Alias for backwards compatibility.
 
 
-class LazyMapping(Mapping[str, str], ABC):
+class LazyMapping(Mapping[str, Union[bool, int, float, str]], ABC):
```

Comment:
That's a good point. I'll see if I can rework it to have LazyMapping be Any and the two subclasses specific.

Reply:
This was a great suggestion - I like it much more now.

It would be even nicer if `LazyMapping` was generic rather than introducing the parent class, but since `ops.LazyMapping` is public it is possible that someone is using it directly (no charms that I have locally do, but you never know) and they'd need to change to `LazyMapping[str]` to have static typing pass.